appsTransfer/src/main/java/org/kinecosystem/appstransfer/presenter/SenderServiceBinder.kt

```diff
@@ -23,8 +23,8 @@ class SenderServiceBinder(private val context: Context?) {
         fun onBalanceFailed()
         fun onServiceConnected()
         fun onServiceDisconnected()
-        fun onTransferFailed()
-        fun onTransferComplete()
+        fun onTransferFailed(errorMessage: String, senderAddress: String)
+        fun onTransferComplete(kinTransferComplete: SendKinServiceBase.KinTransferComplete)
```

Comment:
you have another PR that changes the exact same things ? 
As I wrote to the other PR you should get the latest from master into your feature branch and side branches.

Reply:
yes i rebase the other pr that fix that